doc/user_guide/user-guide.md

```diff
@@ -109,3 +111,13 @@ has already been uploaded one can use the `--no-upload-container` option to skip
 By default, overriding language activation is not permitted. If a language with the same alias has already
 been activated the command will result in an error. The activation can be overridden with the use of
 the `--allow-override` option.
+
+## BucketFS connection object
+
+Some extensions require the user to create a BucketFS connection object encapsulating the BucketFS credentials.
+The creation of such an object in the database would also be a part of the extension installation. Like the
+language container, the connection object can be created separately, using the same or a different command.
+Please check the documentation of a particular extension for details.
+
+Most of the options listed above in the Language Container Deployer sections are also relevant for the creation
+of the connection object, should it be performed in a separate command.
```

Comment:
Should we append a customization section here?
For developers using the PEC to create a custom SLC deployer.
E.g. [AAF](https://github.com/exasol/advanced-analytics-framework/blob/main/exasol_advanced_analytics_framework/deploy.py) or [TE](https://github.com/exasol/transformers-extension/blob/main/exasol_transformers_extension/deploy.py)?

Reply:
This the document for an end user.